README.md

```diff
@@ -75,9 +75,8 @@ production release here:
 The follow described the major aspects of the project structure:
 
 - `azure-pipelines/` - CI configuration files for Azure Pipelines.
-- `docs/` - Interface definition and usage documentation.
-- `examples/` - Usage examples.
-- `mbed_tools/` - Python source files.
+- `docs/generated-docs` - Interface definition and usage documentation.
```

Comment:
Please create a new file `.adr-dir` in the root of mbed-tools pointing at the new adr folder. This is so that [the command line tool `adr`](https://github.com/npryce/adr-tools) will still work.

Reply:
Fixed

Reply:
Are we getting rid of `examples/`

Reply:
Ah, looks like it. I don't think that was intentional. Good catch.

Reply:
There is no `examples` folder in the repository. Therefore removed it from README.md